src/plugins/score-plugin-library/Library/LibrarySettings.cpp

```diff
@@ -113,6 +113,9 @@ void Model::firstTimeLibraryDownload()
 
             rescanLibrary();
           },
+      [] (qint64 bytesRecieved, qint64 bytesTotal) {
```

Comment:
what remains is to put that in a label right ?

Reply:
If you wish to show the progress of the library download. Although I thought that just letting the download happen in the background was less distracting. Where would that label go ? In the popup ?

Reply:
Sure !
Should this be for another PR or this one ?

Reply:
yes definitely, let's keep this one as is